src/packages/dialog/dialog.taro.tsx

```
@@ -179,11 +182,13 @@
     )
   }
 
-  const onHandleClickOverlay = (e: any) => {
+  const onHandleClickOverlay = (
+    e: React.MouseEvent<Element, React.MouseEvent>
+  ) => {
     if (closeOnOverlayClick && visible && e.target === e.currentTarget) {
-      const closed = onOverlayClick && onOverlayClick()
-      closed && onClose?.()
-      closed && onCancel?.()
+      const closed = onOverlayClick && onOverlayClick(e)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)